src/Altinn.Platform/Altinn.Platform.Events/Tests/Mocks/RegisterServiceMock.cs

```diff
@@ -52,5 +66,11 @@ private string GetPartiesPath()
             string unitTestFolder = Path.GetDirectoryName(new Uri(typeof(EventsServiceMock).Assembly.CodeBase).LocalPath);
             return Path.Combine(unitTestFolder, @"..\..\..\Data\parties");
         }
+
+        private string GetPartyPath(int partyId)
+        {
+            string unitTestFolder = Path.GetDirectoryName(new Uri(typeof(EventsServiceMock).Assembly.CodeBase).LocalPath);
```

Comment:
eventsServiceMock ref should probably be changed to RegisterServiceMock

Reply:
It is just to find build path. So can be any class in project. But done